multinode-demo/validator.sh

```diff
@@ -94,8 +94,7 @@ rsync_leader_url=$(rsync_url "$leader")
 tune_networking
 
 set -ex
-$rsync --append --verbose --partial --progress --recursive \
-  "$rsync_leader_url"/config/ "$SOLANA_LEADER_CONFIG_DIR"
+$rsync -vPr "$rsync_leader_url"/config/ "$SOLANA_LEADER_CONFIG_DIR"
```

Comment:
why not `--append`?

Reply:
I'm coming to the conclusion that --append and buffered I/O may interact poorly.

I haven't root-caused 910 yet, even after a day of mucking around.  I've seen leaders corrupt their own ledgers and validator.sh corrupting ledgers on restart, but  don't know how it could happen.

The ledger code does some pretty conservative recovery automatically, but this still isn't catching the issue.